libs/documentation/src/lib/storybook/formly/formly-text-area/formly-text-area-configurable/formly-text-area-configurable.component.ts

```diff
@@ -7,13 +7,6 @@ import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
   templateUrl: './formly-text-area-configurable.component.html',
 })
 export class FormlyTextAreaConfigurableComponent {
-  @Input('config')
```

Comment:
Here you actually want to pass a config in. Add this input back in so you can update the config.